src/Helper/MyParcelCollection.php

```diff
@@ -83,7 +83,7 @@ public function getConsignments($keepKeys = true)
     /**
```

Comment:
Zou je in alle bestanden `<?php declare(strict_types=1);` willen plaatsen ipv `<?php`. Misschien een beetje buiten je story. Maar omdat we nu php 7.1 gebruiken is het wel belangrijk dat we dit gelijk aan het begin doen. Het kan zijn dat er in Magento 2 dingen stuk gaan, maar dat laat dan zien dat het daar niet goed geprogrammeerd is.

Reply:
Done